packages/sdk/package.json

```diff
@@ -0,0 +1,114 @@
+{
+    "name": "@ripio/sdk",
+    "version": "1.1.0",
+    "description": "SDK to interact with Smart Contracts and wallets",
+    "files": [
+      "dist"
+    ],
+    "main": "dist/cjs/index.js",
+    "types": "dist/types/index.d.ts",
+    "exports": {
+      ".": {
+        "types": "./dist/types/index.d.ts",
+        "require": "./dist/cjs/index.js",
+        "import": "./dist/esm/index.js",
+        "default": "./dist/esm/index.js"
+      },
+      "./managers": {
+        "types": "./dist/types/managers/index.d.ts",
+        "require": "./dist/cjs/managers/index.js",
+        "import": "./dist/esm/managers/index.js",
+        "default": "./dist/esm/managers/index.js"
+      },
+      "./connectors": {
+        "types": "./dist/types/connectors/index.d.ts",
+        "require": "./dist/cjs/connectors/index.js",
+        "import": "./dist/esm/connectors/index.js",
+        "default": "./dist/esm/connectors/index.js"
+      },
+      "./utils": {
+        "types": "./dist/types/utils/index.d.ts",
+        "require": "./dist/cjs/utils/index.js",
+        "import": "./dist/esm/utils/index.js",
+        "default": "./dist/esm/utils/index.js"
+      },
+      "./types": {
+        "types": "./dist/types/types/index.d.ts",
+        "require": "./dist/cjs/types/index.js",
+        "import": "./dist/esm/types/index.js",
+        "default": "./dist/esm/types/index.js"
+      },
+      "./gaming": {
+        "types": "./dist/types/gaming/index.d.ts",
+        "require": "./dist/cjs/gaming/index.js",
+        "import": "./dist/esm/gaming/index.js",
+        "default": "./dist/esm/gaming/index.js"
+      },
+      "./ethers-utils": {
+        "types": "./dist/types/utils/ethersUtils.d.ts",
+        "require": "./dist/cjs/utils/ethersUtils.js",
+        "import": "./dist/esm/utils/ethersUtils.js",
+        "default": "./dist/esm/utils/ethersUtils.js"
+      },
+      "./nft/storage": {
+        "types": "./dist/types/nft/storage/index.d.ts",
+        "require": "./dist/cjs/nft/storage/index.js",
+        "import": "./dist/esm/nft/storage/index.js",
+        "default": "./dist/esm/nft/storage/index.js"
+      }
+    },
+    "typesVersions": {
+      "*": {
+        ".": [
+          "dist/types/index.d.ts"
+        ],
+        "managers": [
+          "dist/types/managers/index.d.ts"
+        ],
+        "connectors": [
+          "dist/types/connectors/index.d.ts"
+        ],
+        "utils": [
+          "dist/types/utils/index.d.ts"
+        ],
+        "types": [
+          "dist/types/types/index.d.ts"
+        ],
+        "gaming": [
+          "dist/types/gaming/index.d.ts"
+        ],
```

Comment:
missing storage, and should the gaming one be deleted?

Reply:
Storage is not part of the main package (sdk) anymore. It is on the nft package.